jcl/src/openj9.cuda/share/classes/com/ibm/cuda/CudaPermission.java

```diff
@@ -1,4 +1,4 @@
-/*[INCLUDE-IF Sidecar18-SE]*/
+/*[INCLUDE-IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
I don't think we can simply remove this class. It can be referenced in Java `.policy` files. Instead of reverting this line, please update it to
```java
/*[INCLUDE-IF JAVA_SPEC_VERSION >= 8]*/
```
Then mark the class deprecated for removal
```java
/*[IF JAVA_SPEC_VERSION >= 24]*/
@Deprecated(since = "24", forRemoval = true)
/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
```
(don't forget javadoc).